src/ReleaseHistory.md

```diff
@@ -1,5 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fix the issue `merge` command generates SARIF with multiple runs even specified `--merge-runs` option [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
+* BUGFIX: Fix `System.IO.IOException` and `System.IO.DirectoryNotFoundException` when merge Sarif files contains invalid file name characters in the rule id with `--split PerRule` option.
```

Comment:
I don't see any other references to these exceptions in this change? Was this broken out into one of your other PRs? Or was this included and not documented in a closed PR?

Reply:
This error was not reported by any issue, but found during my testing for `merge` command. My other 2 PRs are for `match-result-forward` command. Its a straightforward fix, so I think it can be fixed along with the multiple runs issue.

Reply:
Can you point me to the specific test implemented to prevent regression?

Reply:
sure, in this test `MergeCommandUnitTests.MergeCommand_WhenSplitPerRule_LogShouldAggregateByRuleToolVersion', it creates sarif with id contains invalid char '/', e.g. 'TESTRULE/001', and the fix replaced the invalid char and generate valid file name e.g. 'TESTRULE.001_merged.sarif'